### ISSUE TITLE
Browserslist: caniuse-lite is outdated. Please run next command `yarn upgrade`

### ISSUE THREAD

**Issue by bonniesimon** [vercel/next.js#15982]
# Bug report

## Describe the bug
When running `yarn dev` the following error pops up.
```
Browserslist: caniuse-lite is outdated. Please run next command `yarn upgrade`
SyntaxError: JSON5: invalid end of input at 1:1
    at syntaxError (/mnt/d/Coding/projects/nextjs-blog/node_modules/next/dist/compiled/json5/index.js:1:28937)
    at invalidEOF (/mnt/d/Coding/projects/nextjs-blog/node_modules/next/dist/compiled/json5/index.js:1:28324)
    at Object.start (/mnt/d/Coding/projects/nextjs-blog/node_modules/next/dist/compiled/json5/index.js:1:26739)
    at Object.parse (/mnt/d/Coding/projects/nextjs-blog/node_modules/next/dist/compiled/json5/index.js:1:19805)
    at parseJsonFile (/mnt/d/Coding/projects/nextjs-blog/node_modules/next/dist/build/webpack-config.js:1:3388)
    at getBaseWebpackConfig (/mnt/d/Coding/projects/nextjs-blog/node_modules/next/dist/build/webpack-config.js:8:193)
    at async Promise.all (index 0)
    at async HotReloader.start (/mnt/d/Coding/projects/nextjs-blog/node_modules/next/dist/server/hot-reloader.js:14:1788)
    at async DevServer.prepare (/mnt/d/Coding/projects/nextjs-blog/node_modules/next/dist/server/next-dev-server.js:10:1565)
    at async /mnt/d/Coding/projects/nextjs-blog/node_modules/next/dist/cli/next-dev.js:21:359 {
  lineNumber: 1,
  columnNumber: 1
}
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

## To Reproduce
run `yarn dev`

## yarn list
[yarn list](https://drive.google.com/file/d/17KRM4AKSSDDvV1jjZukDKbh6tRPX29LM/view?usp=sharing)

## Expected behavior

local server starts up.

## System information

- OS: WSL on windows 10 2004
- Version of Next.js: 9.3.5
- Version of Node.js: 12.8.2

**Comment by jamesmosier** [comment]
Could you provide a reproduction for this?

**Comment by bonniesimon** [comment]
> 
> 
> Could you provide a reproduction for this?

I'm sorry but I don't know what that means. Could you explain to me on how to give proper reproduction for this?

**Comment by timneutkens** [comment]
A github repository that we can look at and run.

**Comment by Timer** [comment]
There's two things in this issue:

> Browserslist: caniuse-lite is outdated. Please run next command `yarn upgrade`

This is a harmless warning that can only be fixed by upgrading Next.js.

> SyntaxError: JSON5: invalid end of input at 1:1

This error is caused by an invalid JSON file in your project. New Next.js versions will now show you the file name and stack trace instead of emitting this unactionable error (already fixed).

**Comment by Timer** [comment]
This was fixed in `next@^9.5.2-canary.14`. Please upgrade and let us know!

**Comment by balazsorban44** [comment]
This issue has been automatically locked due to no recent activity. If you are running into a similar issue, please create a new issue with the steps to reproduce. Thank you.
